src/components/User/DetailsCard.tsx

```diff
@@ -1,28 +1,33 @@
 import css from "./DetailsCard.module.css";
 interface dummyDataProps {
-  hoveredUserProfile: { id: number; pic: string; name: string; email: string }[];
-  // hoveredUserProfile: any;
-  
+  hoveredUserProfile: {
+    pic: string;
+    name: string;
+    email: string;
+    status?: string;
+    access?: string;
+  } | null;
 }
-function DetailsCard({ hoveredUserProfile:[pic:string]}: dummyDataProps) {
+
+function DetailsCard({ hoveredUserProfile }: dummyDataProps) {
   return (
     <div className={css["card"]}>
       <img
         className={css["profile_img"]}
-        src={hoveredUserProfile.pic}
+        src={hoveredUserProfile!.pic}
```

Comment:
Type assertions using `!` are not safe in react based apps.
It can lead to runtime app crashes.
It is suggested to eliminate `null` / `undefined` types by adding a type guard using if check.